src/main/java/org/jboss/xavier/analytics/functions/HelperFunctions.java

```diff
@@ -34,7 +34,7 @@ public static boolean isUnsupportedOS(String osToCheck)
 
     public static boolean isUndetectedOS(String osToCheck)
```

Comment:
To be honest, if all checks are positive it would be clearer to understand.   
This one and the previous method , it's super confusing a negative `!` and a Undetected so that means a detected one.  
wdyt ?

Reply:
You are right, the double negative is a little bit confusing. However, the categorzation methods have been written to match the categories themselves: so as we can categorize an OS as Supported, Unsupported, Convertible or Undetected, we have corresponding methods. The others all have defined criteria to decide if they are true or not, but as far as I'm aware  isUnsupported is merely the absence of any of the other conditions so this method evaluates as true if all the others don't.

Rather than change the names of the methods and make them confusing elsewhere, I think the best solution is for me to comment isUnsupportedOS and explain this. Does that sound ok?

Reply:
Makes sense. I agree :+1: